saltstack/base/salt/selinux/etc/selinux/cdp/ipaldapagent/cdp-ipaldapagent.restorecon

```diff
@@ -0,0 +1,9 @@
+/usr/lib/systemd/system/cdp-freeipa-ldapagent.service
+/etc/selinux/cdp/ipaldapagent-python-wrapper.sh
+/cdp/ipaldapagent/cdp-freeipa-ldapagent
+/cdp/ipaldapagent/wsgi.py
+/cdp/ipaldapagent/gunicorn.conf.py
+/cdp/ipaldapagent/devserver.log.conf
+/cdp/ipaldapagent/gunicorn.log.conf
+/cdp/ipaldapagent/requirements.txt
+/cdp/ipaldapagent/NOTICE.txt
```

Comment:
Just adding `/cdp/ipaldapagent` by itself should be enough, as we call restorecon with the `-R` flag, which will recursively apply the file contexts to all files under the directory.